lib/pbench/test/unit/server/query_apis/test_iteration_samples.py

```diff
@@ -679,3 +683,23 @@ def test_iteration_samples_with_scroll_id(
             assert response.json == {
                 "results": [hit["_source"] for hit in response_payload["hits"]["hits"]]
             }
+
+    def test_exceptions_on_get_index(self, attach_dataset):
```

Comment:
Now you've taken the first step: how hard would it be to add some non-failure cases now?? 😈

Reply:
Ah, yeah I could have added the non-failure cases here, but they should already be indirectly tested under other tests that check the entire query api.